fe/fe-core/src/main/java/org/apache/doris/datasource/hive/HiveMetaStoreCache.java

```diff
@@ -984,6 +1022,43 @@ public void setAcidInfo(AcidInfo acidInfo) {
         }
     }
 
+    @Data
+    public static class FileSystemCacheKey {
```

Comment:
I think this class should be implement in `fs` package.
It is not only for `HiveMetaStoreCache`